### DISCUSSION TITLE
Why is the sd-auto image so large?

### DISCUSSION THREAD

**Discussion by ca1123** [AbdBarho/stable-diffusion-webui-docker#394] [Q&A]
**Describe the bug**

The push refers to repository [harbor.kube1.jupiter/ju-container-registry/sd-auto]
5f70bf18a086: Pushed 
0bbf094f7cb3: Pushed 
adab82e6c017: Pushed 
7bdef3319887: Pushed 
f4266e5968ae: Pushed 
a81d1fb51f4f: Pushed 
f1711be1ef88: Pushed 
76b729e42e50: Pushed 
ab79bddb9e78: Pushed 
53c16538a383: Pushed 
88cbd5cfe000: Pushed 
3404d605e8ea: Pushed 
d548487982be: Pushed 
b559dd225cf3: Pushing [============================================>      ]  3.107GB/3.474GB
2e0e2e9caf0d: Pushed 
622a500a957f: Pushed 
99fd976ad0a9: Pushed 
10b4939044cd: Pushed 
bd2fe8b74db6: Pushed 

**Which UI**

auto

**Hardware / Software**

not really matter

**Steps to Reproduce**

Any succesfully ran docker composed image has this problem.

**Additional context**
I am adapating this thing to run on my k8s cluster.

**Comment by AbdBarho** [comment]
pytorch by itself is more than 2GB.

the cloned repo is also hundreds of MBs
